src/Symfony/Component/Scheduler/Trigger/CronExpressionTrigger.php

```diff
@@ -73,9 +76,26 @@ public static function fromSpec(string $expression = '* * * * *', string $contex
         return new self(new CronExpression(self::parseHashed($expression, $context)));
     }
 
+    public function jitter(int $value = 60): self
```

Comment:
I kept this as a _builder_ method but can add as a constructor argument if desired.

Reply:
Just wondering if we could rename the method argument from `$value` to `$seconds` or `$numOfSeconds` or something that makes it self-explanatory. Thanks.

Reply:
Good idea. I've renamed to `$maxSeconds`.